modules/core/client/app/init.js

```diff
@@ -12,7 +12,23 @@ angular.module(ApplicationConfiguration.applicationModuleName).config(['$locatio
   }
 ]);
 
-angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, Authentication) {
+angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, $http, Authentication, CORE_CONST) {
+
+  //set token header
+  var auth_token = localStorage.getItem('auth_token');
```

Comment:
@jloveland Thank you for bringing up that point. It's definitely something that we should discuss.

I did read, however, that since the token's are sent with the Authorization header it's not vulnerable to CSRF because CSRF doesn't include the header. And I may be misinterpreting this statement from this article; I loosely based my implementation off this btw 
http://code.tutsplus.com/tutorials/token-based-authentication-with-angularjs-nodejs--cms-22543
>No Cookie-Session (or No CSRF). CSRF is a major problem in modern web security because it doesn't check whether a request source is trusted or not. To solve this problem, a token pool is used for sending that token on every form post. In token-based authentication, a token is used in authorization headers, and CSRF does not include that information.